src/PrestaShopBundle/Resources/views/Admin/macros.html.twig

```diff
@@ -39,7 +39,7 @@
 {% macro infotip(text)%}
 <div class="alert alert-info" role="alert">
   <p class="alert-text">
-    {{ text }}
+    {{ text|raw }}
```

Comment:
Any reason to use raw here?

Reply:
It is so that html is interpreted, otherwise when our translated message contains a link, that is to say a html tag (<a href ...) it will be displayed without being interpreted.

Reply:
@PierreRambaud would you prefer something like this?

`{{ text|striptags('<a>') |raw}}`

So that only <a> tags are accepted ?

Reply:
@PierreRambaud  up :)

Reply:
I think he would prefer no `raw` at all 😄

Do you know which usecases require to display such links ? It would be better to treat them as edge-cases with dedicated process. Something like:

```
{% if message.is_regular %}
    {{ text }}
{% elseif message.has.link %}
    {{ display_message_with_link(message) }}
{%  endif %}
```

Reply:
If you change the behaviour by adding `raw`, you must be sure all variables using this macro must be escaped.
Good idea from @matks :+1:

Reply:
The idea of `{{ text|striptags('<a>')|raw}}`was that striptags escapes everything except `<a>` tags, and the raw makes sure the non escaped "a" tag is interpretated. 

So all variables using this macro would be escaped (except their <a> tags).

But I like @matks 's solution too, I will do that.  👍

Reply:
🤔I dont think scripttags is reliable
Here what we want to avoid are XSS vulnerabilities, and you can encode tags in a looooot of different ways. I dont think scripttags removes them all

Reply:
See https://www.owasp.org/index.php/XSS_Filter_Evasion_Cheat_Sheet

Reply:
@matks @PierreRambaud 
Ok I made a modification: instead of detecting if the link has a `<a>`tag or not, I made it possible to pass allowed tags as a parameter in the infotip macro function, if no allowed tag is specified then raw is not used at all.

What do you think?

Reply:
damn I just saw your previous message @matks

Reply:
Striptags can remove unwanted characters

```
Hi, <:) we found it
```
will display
```
Hi,
```

Do we really need to add a link in the infotip?
Or do we want to add create a special infotip which is not escape? Maybe a `infotipraw`

Reply:
@PierreRambaud 

Well the link was not asked at the beginning, but suggested in this PR during the code review.
I do think it is useful for the end user though.

Do you mind if I add a raw parameter to the infotip function, whose default value is false, so that we don't duplicate the html inside the macro ?

Reply:
(that's what I did)

Reply:
If the link was suggested, I'm more in favor to remove it and keep the original behavior.
Otherwise if it's mandatory, adding a second parameter like you said if a good idea :).
At least it will be use only if necessary